playwright-e2e/tests/singular/enrollment/self-enrollment.spec.ts

```diff
@@ -0,0 +1,166 @@
+    await aboutYourself.haveVentricleHeartDefect().checkbox('Yes').check();
+    // Checkbox "I'm not a robot"
+    await aboutYourself.checkReCaptcha();
+    await aboutYourself.signMeUp.click();
+
+    // Enter email alias and new password in Login popup
+    await fillEmailPassword(page, {
+      email: makeEmailAlias(process.env.singularUserEmail as string),
+      password: process.env.singularUserPassword,
+      waitForNavigation: true
+    });
+
+    // On "My Dashboard" page
+    const myDashboardPage = new MyDashboardPage(page);
+    await myDashboardPage.waitForReady();
+    await myDashboardPage.enrollMyself.click();
+
+    // On "Enroll myself" page
+    await assertActivityHeader(page, 'Enroll myself');
+    await myDashboardPage.whoHasVentricleHeartDefect(WHO.Me).check();
+    await myDashboardPage.next.click();
+
+    // On "Consent Form" page
+    const consentForm = new ConsentFormPage(page);
+    await consentForm.waitForReady();
+    await assertActivityHeader(page, 'Your Consent Form');
+    await assertActivityProgress(page, 'Page 1 of 3');
+    await consentForm.next.click();
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await consentForm.next.click();
+    await assertActivityProgress(page, 'Page 3 of 3');
+    await consentForm.firstName().fill(user.patient.firstName);
+    await consentForm.lastName().fill(user.patient.lastName);
+    await consentForm.enterDateOfBirth(12, 20, 1950);
+    await consentForm.wantToKnowSecondaryFinding('I want to know.').check();
+    await consentForm.signature().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await consentForm.authorizationSignature().fill(user.patient.lastName);
+    await consentForm.agree();
+
+    // on "About Me" page
+    const aboutMePage = new AboutMePage(page);
+    await aboutMePage.waitForReady();
+    await assertActivityHeader(page, 'About Me');
+    await aboutMePage.fullName().locator.fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await aboutMePage.country(user.patient.country.name);
+    await aboutMePage.street().locator.fill(user.patient.streetAddress);
+    await aboutMePage.city().locator.fill(user.patient.city);
+    await aboutMePage.state(user.patient.state.name);
+    await aboutMePage.zipCode().locator.fill(user.patient.zip);
+    await aboutMePage.telephone().locator.fill(user.patient.phone);
+    await aboutMePage.next.click();
+    // Trigger validation
+    await aboutMePage.pickSuggestedAddressEntry('As Entered:');
+    await Promise.all([page.waitForNavigation(), aboutMePage.next.click()]);
+
+    // on "Medical Record Release Form" page
+    const medicalRecordReleaseForm = new MedicalRecordReleaseForm(page);
+    await assertActivityHeader(page, 'Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 1 of 3');
+    await medicalRecordReleaseForm.unableToProvideMedicalRecords().check();
```

Comment:
I've got the following:
![image](https://user-images.githubusercontent.com/109761417/194152282-a1613089-5fa0-4139-9004-5db9aba93f71.png)

Reply:
This is what looked like now in my Intellij. Highlighted functions in your screenshot have not been removed. What IDE are you using? Do you think it might be a IDE setting?
<img width="689" alt="Screen Shot 2022-10-05 at 4 34 24 PM" src="https://user-images.githubusercontent.com/35533885/194158392-979b21de-606a-4ba8-a266-c1f889f11e64.png">

Reply:
Just to be sure, you've pulled down latest code, right?